js/listNumbers.js

```diff
@@ -0,0 +1,33 @@
+var decimalBullet = Array.apply(null, Array(100)).map(function (a, i) {
```

Comment:
any reason not to use Array.from here rather than spreading a sparse array?

Reply:
This file is written in ES5. It's probably a good idea to switch over to more modern JS, but I don't want to that as part of this PR.

Reply:
This is a brand new file - do you mean the other files are written in ES5, and you wanted this one to match?

Reply:
Sorry, yes, that.